api/src/main/java/jakarta/enterprise/util/AnnotationLiteral.java

```diff
@@ -38,6 +38,7 @@
  * 
  * <p>
  * An instance of an annotation type may be obtained by subclassing <code>AnnotationLiteral</code>.
+ * The subclass must implement the annotation interface to satisfy the {@link Annotation} contract.
```

Comment:
```suggestion
 * The subclass must implement the Annotation interface to satisfy the {@link Annotation} contract.
```

Reply:
One thing I could do is replace "annotation interface" with "annotation type", as "annotation type" is used above this sentence. However, "the subclass must implement the annotation type" is not exactly clear in that "implement" means the `implements` keyword...

Reply:
ah, okay. It makes sense! Thanks for your explanation!